Cargo.toml

```diff
@@ -21,15 +21,14 @@ members = [
     'examples/ffmpeg',
     'examples/gstreamer-player',
     'examples/plotter',
-    'examples/printerdemo_old/rust',
-    'examples/printerdemo/rust',
-    'examples/printerdemo_mcu',
+    'demos/printerdemo/rust',
+    'demos/printerdemo_mcu',
     'examples/slide_puzzle',
     'examples/todo/rust',
     'examples/todo-mvc/rust',
     'examples/virtual_keyboard/rust',
     'examples/carousel/rust',
-    'examples/energy-monitor',
+    'demos/energy-monitor',
     'examples/mcu-board-support',
     'examples/uefi-demo',
     'examples/weather-demo',
```

Comment:
weather-demo, as the same suggest, is another candidate to be moved.

(While uefi-demo is really an example, and not a demo.)

Reply:
Yes. Preferably not in this PR as it's already getting big :)